packages/suite/src/support/messages.ts

```diff
@@ -8933,7 +8933,7 @@ export default defineMessages({
     TR_STAKE_UNSTAKING_TAKES: {
         id: 'TR_STAKE_UNSTAKING_TAKES',
         defaultMessage:
-            'Unstaking currently takes {count, plural, one {# day} few {# days} other {# days}}. Once completed, you can trade or send it.',
+            'Unstaking currently takes {count, plural, one {# day} other {# days}}. Once completed, you can trade or send it.',
```

Comment:
few should be in all strings
because 1 den, 2 dny, 5 dnů

Reply:
It was not working with "few" parameter, Crowdin showed syntax error. Thats why we get rid of ot

Reply:
but it is valid for Czech, so it should me modified there

Reply:
We will take care of it in Crowdin as well